tic_tac_toe.py

```diff
@@ -0,0 +1,227 @@
+    strike of his own three symbol, it will be Tie.
+    Have Fun!"""
+    return rules_open_message
+
+
+def player_get_symbols():  # Each player get symbol.
+    players_symbols_dict = {player_1: x_symbol,
+                            player_2: o_symbol}
+    return players_symbols_dict
+
+
+def random_beginner():  # Randomize the symbols, every game the one who starts, start with "X" object.
+    # This function randomize the object that do the first move.
+    players_symbols_dict = player_get_symbols()
+    beginner = random.choice(list(players_symbols_dict.keys()))
+    second_player = ""
+
+    if beginner == player_1:
+        second_player = player_2
+    elif beginner == player_2:
+        second_player = player_1
+    random_players_symbols = {beginner: x_symbol,
+                              second_player: o_symbol}
+    return random_players_symbols
+
+
+random_beginner_var = random_beginner()
+
+
+def return_key():
+    # Get the key to know who start and who's th second one.
+    # It's not like the first dictionary because we randomize the objects.
+    # Need to know the winner by his symbol.
+    beginner = ""
+    second_player = ""
+    players_dict = random_beginner_var
+
+    count = 0
+    for key in players_dict:
+        count += 1
+        if count == 1:
+            beginner = key
+        if count == 2:
+            second_player = key
+    return [beginner, second_player]
+
+
+return_key_var = return_key()
+
+
+def open_message():
+    # Open message after randomize objects.
+    # Give the players information which symbol each one get.
+    message = ""
+    random_players_symbols = random_beginner_var
+    count = 0
+    for key in random_players_symbols:
+        count += 1
+        if count == 1:
+            message = ("%s is the beginner with %s symbol.\n" % (key, random_players_symbols[key]))
+        elif count == 2:
+            message += ("%s is the second player with %s symbol." % (key, random_players_symbols[key]))
+    return message
+
+
+def check_vertical(board: list):  # Check vertical win.
+    winner = False
+    players_keys_list = return_key_var
+
+    if board[0][0] == board[1][0] == board[2][0]:
+        if board[0][0] == "X":
+            winner = ("the winner is %s." % (players_keys_list[0]))
+        elif board[0][0] == "O":
+            winner = ("the winner is %s." % (players_keys_list[1]))
+
+    if board[0][1] == board[1][1] == board[2][1]:
+        if board[0][1] == "X":
+            winner = ("the winner is %s." % (players_keys_list[0]))
+        elif board[0][1] == "O":
+            winner = ("the winner is %s." % (players_keys_list[1]))
+
+    if board[0][2] == board[1][2] == board[2][2]:
+        if board[0][2] == "X":
+            winner = ("the winner is %s." % (players_keys_list[0]))
+        elif board[0][2] == "O":
+            winner = ("the winner is %s." % (players_keys_list[1]))
+
```

Comment:
If you will use nested for loop instead:
- the code will be kept in less lines
- the code will be more readable
- the function will be generic, as it goes dynamically through the size of the lists instead of hardcoding the numbers --> it shall handle any size you choose in the future for the board, without changing this function again